### DISCUSSION TITLE
Add a way to add/remove a few orbs and skull gems in Debug menu

### DISCUSSION THREAD

**Discussion by Zedb0T** [open-goal/jak-project#3251] [Jak 2]
You can do this with L3 + X on a second controller, but since we have the source it would be possible to make it a little nicer.

**Comment by LuminarLight** [comment]
What would this be used for?

**Reply by Zedb0T** [reply]
It would be used for adding or removing a few orbs or skull gems in the debug menu.

**Comment by Zedb0T** [comment]
Mostly done in #3539
